src/style.css

```diff
@@ -0,0 +1,109 @@
+html,
+body,
+form,
+img,
+div {
+  margin: 0;
+}
+
+@font-face {
+  font-family: "AvenirBook";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "AvenirBold";
+  font-style: normal;
+  font-weight: 400;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.lg-search-bar__form,
+.sm-search-bar__form {
+  position: relative;
+  margin-top: 20px;
+  margin-left: 8px;
+}
+
+.lg-search-bar__icon {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+
+  position: absolute;
+  left: 0;
+  width: 19px;
+  height: 19px;
+  z-index: 1;
+  margin: 25px 17px 26px 26px;
+}
+
+.lg-search-bar__input,
+.sm-search-bar__input {
+  position: relative;
+  z-index: 0;
+  width: 100%;
+
+  font-family: "AvenirBook", serif;
+  color: #3d4e61;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.lg-search-bar__input {
+  max-width: 900px;
```

Comment:
You limit your input to a specific width in pixels, so the width doesn't fully adapt to the width of the form.
The same applies to other forms.